pkg/config/setting.go

```diff
@@ -728,6 +732,19 @@ func applyAttrPatch(attr *PodAttr, setting *JfsSetting) {
 	attr.VolumeMounts = patch.VolumeMounts
 	attr.Volumes = patch.Volumes
 	attr.Env = patch.Env
+
+	// merge options
+	if len(setting.Options) > 0 && len(patch.MountOptions) > 0 {
```

Comment:
what if len(setting.Options) == 0 and len(patch.MountOpions) > 0 ?

Reply:
fixed